.github/workflows/lint.yml

```
@@ -0,0 +1,16 @@
+name: Lint userstyles
```

```suggestion
name: Userstyle Linter
```

This matches with the other workflow names, however, we might want to change the others to match with this PR instead.